config/application.rb

```diff
@@ -18,4 +18,28 @@ class Application < Rails::Application
 
     config.active_job.queue_adapter = :delayed_job
   end
+
+  def self.configure_logging(config)
+    # Move production log into a system folder
+    log_prefix = Rails.env.production? ? "/var/log/exercism/exercism_" : "log/"
+
+    rails_log_file = "#{log_prefix}#{Rails.env}.log"
+    logger         = ActiveSupport::Logger.new(rails_log_file, "daily")
+    config.logger  = ActiveSupport::TaggedLogging.new(logger)
+
+    # Create a lograge event log
+    requests_log_file = "#{log_prefix}requests.log"
+    requests_logger   = ActiveSupport::Logger.new(requests_log_file, "daily")
+    event_logger      = ActiveSupport::TaggedLogging.new(requests_logger)
+
+    config.lograge.keep_original_rails_log = true
+    config.lograge.enabled = true
+    config.lograge.logger = event_logger
+    config.lograge.custom_options = lambda do |event|
+      {
+        remote_ip: event.payload[:remote_ip],
+        request_id: event.payload[:request_id]
+      }
+    end
+  end
```

Comment:
I didn't really know where to put this common concern of config. It's really only needed in the `production.rb` but I wanted to test it out in development mode too. Putting these things in an initialiser doesn't seem to work for me, but maybe it would be better?